.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,149 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for all benchmark-related jobs. It is designed to be called from
+# the main workflow ci.yml. The concurrency of this workflow is controlled by
+# the caller's job.
+
+name: Benchmarks
+
+on:
+  workflow_call:
+    inputs:
+      runner-group:
+        required: true
+        type: string
+      runner-env:
+        required: true
+        type: string
+      gcs-dir:
+        required: true
+        type: string
+      build-dir:
+        required: true
+        type: string
+      build-dir-archive:
+        required: true
+        type: string
+      build-dir-gcs-artifact:
+        required: true
+        type: string
+      tf-binaries-dir:
+        required: true
+        type: string
+      tf-binaries-archive:
+        required: true
+        type: string
+      tf-binaries-gcs-artifact:
+        required: true
+        type: string
+
+jobs:
+  build_benchmark_suites:
+    runs-on:
+      # Hacks, and order matters. See the comment at the top of the ci.yml.
+      - self-hosted
+      - runner-group=${{ inputs.runner-group }}
+      - environment=${{ inputs.runner-env }}
+      - cpu
+      - os-family=Linux
+    env:
+      BUILD_DIR: ${{ inputs.build-dir }}
+      BUILD_DIR_ARCHIVE: ${{ inputs.build-dir-archive }}
+      BUILD_DIR_GCS_ARTIFACT: ${{ inputs.build-dir-gcs-artifact }}
+      TF_BINARIES_DIR: ${{ inputs.tf-binaries-dir }}
+      TF_BINARIES_ARCHIVE: ${{ inputs.tf-binaries-archive }}
+      TF_BINARIES_GCS_ARTIFACT: ${{ inputs.tf-binaries-gcs-artifact }}
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+        with:
+          submodules: true
```

Comment:
Do we actually need the submodules?

Reply:
Looks like there might be some deps to be cleaned up.
https://github.com/iree-org/iree/runs/8256083640

Reply:
Darn. Thanks for checking